src/views/ModelPlan/TaskList/OpsEvalAndLearning/Evaluation/__snapshots__/index.test.tsx.snap

```diff
@@ -492,6 +494,7 @@ exports[`Model Plan Ops Eval and Learning matches snapshot 1`] = `
         class="usa-label maxw-none"
         data-testid="label"
         for="ops-eval-and-learning-data-to-send"
+        id="ops-eval-and-learning-data-to-send"
```

Comment:
I think adding this ID makes us have 2 IDs that are the same on this page: https://github.com/CMSgov/mint-app/blob/6ec9517315c4e0963e7e3fbfa45acb2b6c1b95c8/src/views/ModelPlan/TaskList/OpsEvalAndLearning/Evaluation/index.tsx#L517

I think that's what's breaking the E2E test here https://github.com/CMSgov/mint-app/blob/6ec9517315c4e0963e7e3fbfa45acb2b6c1b95c8/cypress/e2e/opsEvalAndLearning.spec.js#L285-L287


![image](https://user-images.githubusercontent.com/15203744/235460081-753f2aeb-406b-4ecf-a747-969c26f7898d.png)